llama-index-core/tests/workflow/test_event.py

```diff
@@ -0,0 +1,58 @@
+from llama_index.core.workflow.events import Event
+from llama_index.core.bridge.pydantic import PrivateAttr
+from typing import Any
+
+
+class TestEvent(Event):
```

Comment:
nit: pytest will try to run any class named `TestSomething` it finds like a test case and emit a warning. I usually prefix an underscore like `_TestEvent`